### PR TITLE
add coveralls ci

### PR DESCRIPTION
**Pull request by AKidd95**
#### Description

* add coveralls continuous integration 

#### What I have learned working on this feature: 
I learned how to setup coveralls even better than I did before

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ group :development, :test do
 	gem 'database_cleaner'
 	gem 'shoulda-matchers'
   gem 'launchy'
+  gem 'coveralls', require: false
```
Comment (magnus-thor) [inline Gemfile]: Try to fix the indentation.

Comment (magnus-thor) [reply Gemfile]: If it looks good in your ide like it did with antonella, delete both lines and write them again.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+* Cheers for all the help and guidance from the coaches and fellow students at Craft Academy. who help us to learn and grow as developers each and every single day of the camp.
+* Inspiration
+* etc
```
Comment (magnus-thor) [inline README.md]: The readme should have a separate PR.

File: coverage/.resultset.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "RSpec": {
+    "coverage": {
+    },
+    "timestamp": 1519671487
+  }
+}
```
Comment (magnus-thor) [inline coverage/.resultset.json]: Remove coverage files from git.

File: config/coveralls.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+service_name: semaphore
+repo_token: ydfjrASHoqWlKe30gExizi5eubrBh0ZjZ
```
Comment (diraulo) [inline config/coveralls.yml]: You don't want to track sensitive information like tokens in version control.

**Review by diraulo — CHANGES_REQUESTED** [review]
You actually don't need to have this `coveralls.yml` file here. You should add this token as an environment variable on Semaphore.

File: .gitignore [hunk 5]
```diff
@@ -18,3 +18,6 @@
 
 .byebug_history
 .idea
+
+/coverage.data
```
Comment (diraulo) [inline .gitignore]: replace these 2 lines with `coverage` it should take care of excluding the entire folder

**Comment by diraulo** [comment]
Okay I see you've added Coveralls to the story on PT. You should probably change the title of this PR to reflect what you're doing. eg `Setup continuous integration and coverage report`

As for Coveralls you're only setting it up for RSpec. What about Cucumber? And also you should probably merge both reports

**Comment by magnus-thor** [comment]
Looks good, good job.

**Pull request merged**